pom.xml

```diff
@@ -3,29 +3,33 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.480.3</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>2.9</version>
   </parent>
-
+  
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>build-view-column</artifactId>
   <version>0.3-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
   <name>Build View Column Plugin</name>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/Build+View+Column+Plugin</url>
+  
+  <properties>
+    <jenkins.version>1.609.3</jenkins.version>
```

Comment:
> The 1.480.3 line is no longer found in the tagged releases in GitHub

What?!

There are known problems with some older lines when running tests on Java 8, which would merit an update, but not as far as 1.609.x. 1.580.x is usually considered uncontroversial.